test_crates/declarative_macro_missing/new/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+macro_rules! will_no_longer_be_exported {
+    () => {};
+}
```

Comment:
Adding some more test cases for `#[doc(hidden)]` on macro-rules macros.

Also, highly recommend configuring your editor to automatically add trailing newlines at the end of all files, they are necessary to keep shell tools like `wc` working correctly since they assume all lines end in `\n`.
```suggestion
}

#[doc(hidden)]
#[macro_export]
macro_rules! became_doc_hidden {
    () => {};
}

#[doc(hidden)]
#[macro_export]
macro_rules! always_doc_hidden {
    () => {};
}

```